int64 motor_no
float64 val
---
bool success
